assignments/prototypes.js

```diff
@@ -16,13 +16,34 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+function GameObject(attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+  this.destroy = function() {
```

Comment:
the instructions was to add a `destroy()` function to the `GameObject`'s prototype.  You manually added it as a property to `GameObject`.  It works, but we want you to practice adding it to the prototype.